README.md

```diff
@@ -6,6 +6,17 @@ A service to return older system profile records
 (need to fill this in)
  * make sure you have `libpq-devel` installed
 
+## db changes and migration
+
+The db schema is defined by the objects defined in models.py.  When a change is made to these model objects, a database migration needs to be created.  This migration will be applied automatically when an updated image is spun up in a pod.  The steps to create the database migration are below:
+
+* make changes to model objects in models.py
+* in the system-baseline-backend source directory, run `pipenv shell`
```

Comment:
in the historical-system-profiles source directory

Reply:
Derp!  Cut-n-paste miss.  Force-pushed an update.